src/modules/Extension/html_admin/mod_extension_index.html.twig

```diff
@@ -203,5 +213,8 @@
                 bb.reload();
             }
         }
+        function onAfterModuleUninstalled() {
```

Comment:
This shouldn't be needed.
You should be able to add `data-api-reload="1"` to have the page reload